src/systemtest/java/com/ericsson/ei/systemtest/utils/Config.java

```diff
@@ -39,4 +42,22 @@ public void initRemRemConfig() {
 
         assertNotNull(remremBaseUrl);
     }
+
+    /**
+     *
+     * This method retrieves the EI front end properties from the system properties.
+     *
+     * @return
+     */
+    public void initEIFrontend() {
+        eiFrontendBaseUrl = System.getProperty("EI_FRONTEND_BASE_URL");
+
+        assertNotNull(eiFrontendBaseUrl);
+    }
+
+    public void initEIBackend() {
+        eiBackendBaseUrl = System.getProperty("EI_BACKEND_BASE_URL");
+
```

Comment:
Same comment here.

Reply:
It should be changed to that in this PR :)